coda.rb

```diff
@@ -16,6 +17,42 @@ class Coda < Formula
   conflicts_with "coda-dev"
 
   bottle :unneeded
+  
+    plist_options :manual => "coda"
+
+  def plist; <<~EOS
+    <?xml version="1.0" encoding="UTF-8"?>
+    <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+    <plist version="1.0">
+    <dict>
+      <key>Label</key>
+      <string>#{plist_name}</string>
+      <key>ProgramArguments</key>
+      <array>
+        <string>#{bin}/coda</string>
+        <string>daemon</string>
+        <string>-peer</string>
+        <string>/dns4/seed-one.genesis-redux.o1test.net/tcp/10002/ipfs/12D3KooWP7fTKbyiUcYJGajQDpCFo2rDexgTHFJTxCH8jvcL1eAH</string>
+        <string>-peer</string>
+        <string>/dns4/seed-two.genesis-redux.o1test.net/tcp/10002/ipfs/12D3KooWL9ywbiXNfMBqnUKHSB1Q1BaHFNUzppu6JLMVn9TTPFSA</string>
```

Comment:
It might be a good idea to make these variables.

Reply:
@Xe I tried making the variables in the formula, not sure if that's what you meant (at least it's a little easier to change them now)

Reply:
This is a great idea! Given we'll have to change these probably most of the time we increment that version integer, it's good to colocate those